SPECS/containerd2/containerd.toml

```diff
@@ -0,0 +1,25 @@
+#root = "/var/lib/containerd"
```

Comment:
Why do we need the commented stuff? :) Is this only ported from the prior SPEC or is this actively being utilized in some cases?

Reply:
This commented part is from the existing containerd package, we can remove them for containerd2 package.